guides/v2.1/config-guide/varnish/config-varnish-configure.md

```diff
@@ -199,7 +199,7 @@ A long list of response headers display in your command prompt window. Look for
 	-   VCL_call       BACKEND_FETCH
 	-   VCL_return     fetch
 	-   BackendOpen    17 default(10.249.151.10,,8080) 10.249.151.10 60914
-	-   Backend        17 default default(10.249.151.10,,8080)
+	-   Backend        17 default(10.249.151.10,,8080)
```

Comment:
@tejash-wagento, Hi
These changes are already fixed and accepted. in this PR: https://github.com/magento/devdocs/pull/2448